src/types/coursDetails.type.ts

```diff
@@ -0,0 +1,9 @@
+export type coursDetails = {
+    id: number,
+    title: string,
+    level: string,
+    description: string,
+    rating: number,
+    nbrOfRaters: number,
+    trainer: string
+}
```

Comment:
i think languages is missing
```suggestion
export enum Language {
  ARABIC = 'arabic',
  ENGLISH = 'english',
}
....
languages: Language[],
}
```